noether_tpp/include/noether_tpp/tool_path_modifiers/straight_approach_modifier.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <noether_tpp/core/tool_path_modifier.h>
+
+namespace noether
+{
+/**
+ * @brief Modifier that adjusts the parameters of the tool approach trajectory to the media
+ */
+class StraightApproachModifier : public ToolPathModifier
+{
+public:
+  StraightApproachModifier(double offset_height_, double n_points);
+  ToolPaths modify(ToolPaths tool_paths) const override final;
+
+protected:
+  const double offset_height_;
```

Comment:
Let's make this more flexible by allowing the user to choose to offset in any arbitrary direction. There may be cases where we want to approach from the x- or y-axis rather than along the normal

```suggestion
  Eigen::Vector3d offset_;
```

Reply:
![pr_widget](https://github.com/ros-industrial/noether/assets/76972296/e97abcef-dec4-495a-bc80-cc342ae7bdfd)
Modified widget to offset the starting point of each raster in the direction the user specifies.